bulk-model-sync-solution/build.gradle.kts

```diff
@@ -0,0 +1,42 @@
+import org.modelix.gradle.mpsbuild.MPSBuildSettings
+
+plugins {
+    base
+    alias(libs.plugins.modelix.mps.buildtools)
+}
+
+group = "org.modelix.mps"
+
+val generatorLibs: Configuration by configurations.creating
+
+dependencies {
+    "generatorLibs"(project(":bulk-model-sync-lib"))
+    generatorLibs(project(":mps-model-adapters"))
```

Comment:
Why are two different styles used here?

Reply:
This must have been caused by IntelliJ while renaming the component.